Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14159,6 +14261,21 @@ end
 					},
 					Notes = "Returns the current OS time, as a unix time stamp (number of seconds since Jan 1, 1970)",
 				},
+				IsBiomeCold =
+				{
+					Params =
+					{
+						{
+							Name = "Biome",
+							Type = "EMCSBiome",
+						},
+					},
+					Returns =
+					{
+						{ Type = "boolean" },
+					},
+					Notes = "Returns true if the biome is cold. Doesn't report very cold biomes, use IsBiomeVeryCold() for those.",
```

Comment:
Next try :-)
`Returns true if the biome is cold, it has rain and at certain height it snows. The biome's are mostly of type high hills and taiga`